examples/models/metadata/example-01/Model.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import json
+import os
+
+
+class Model:
+    def predict(self, features, names=[], meta=[]):
+        logging.info(f"model features: {features}")
+        logging.info(f"model names: {names}")
+        logging.info(f"model meta: {meta}")
+        return features.tolist()
+
+    def init_metadata(self):
+        logging.info("metadata method  called")
+
+        meta = {
+            "name": "my-model-name",
+            "versions": ["my-model-version-01"],
+            "platform": "seldon-custom",
+            "inputs": [{"name": "input", "datatype": "BYTES", "shape": [1]}],
```

Comment:
why is it bytes?

Reply:
"BYTES" would basically mean strings - need to check though if this is what I am sending to the model in this example

Reply:
Oh right, I remember you mentioned that, just double checked on the api v2 kfserving dataplane